tests/product/test_control.py

```diff
@@ -105,67 +78,24 @@ def test_server_restart_nothing_started(self):
             self.expected_start()[:]
         self.assert_simple_server_restart(expected_output, running_host='')
 
-    def test_server_restart_coordinator_started(self):
```

Comment:
The two tests below can be removed because `test_server_stop_node_started()` performs an action when a node is already up.